testing/php7-apcu/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Maintainer: Carlo Landmeter <clandmeter@gmail.com>
+pkgname=php7-apcu
+_pkgreal=apcu
```

Comment:
Please name it rather as `_pkgname`, `_pkgreal` has very short Levenshtein distance from `_pkgrel`. ;)